### PR TITLE
Hello Microverse project

### PR DESCRIPTION
**Pull request by rex-9**
Added Linters and README
Added html and css files

### CODE REVIEW

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
- [ ] Kindly use [this template](https://github.com/microverseinc/readme-template) to make you README.md file descriptive, but remember to customize it to your project.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Hello-World
\ No newline at end of file
+Project Name - Hello Microverse
+Description - Hello World html template with red color.
+Built with - HTML and CSS with linters
```
Comment (zieeco) [inline README.md]: - [ ] **[REQUIRED CHANGES]**

- Hey, great fixing the changes required by the previous reviewer, however, you've not add all the necessary headers to your readme file to make it look professional. kindly do the following:
-  Copy and paste the content of this [microverse readme template](https://github.com/microverseinc/readme-template) into your readme file.
- change the `Project Name` to `Hello Microverse`
- Add a little description of the project. like the aim of the project
- Remove the `Live Demo (if available)` for now, except you know how to use the `gh` pages to add live link. You will learn this later in the curriculum
- In the `Builth With` header, Add this:
```

- HTML
- CSS
```
- In the `Getting Started` header, add the following:
```

To get a local copy up and running:

1. Clone this repository or download the zip folder:

**``git clone https://github.com/HtetNaing0814/Hello-World.git``**

1. Navigate to the location of the folder in your machine:

**``you@your-Pc-name:~$ cd <folder>``**

```
- In the `Authors` header, change it to be singular, and the add replace the links to your personal detail in the social media networks..  Here:

```
## Author

👤 **your name**

- Github: [@your nme](your GitHub link)
- Twitter: [@your name](your Twitter link) # if and only if you have one
- LinkedIn: [@your name](your Linkedin link) # if and only if you have one
```
- In the `Contribution header` change the issue link to match the issue link of your project like this 
```
Feel free to check the [issue page](https://github.com/HtetNaing0814/Hello-World/issues)
```

- Update the `Acknowledgement` header by adding the microverse official link:
```
 [Microverse](https://bit.ly/MicroverseTN)
```
- In the `License` header add the content in the `MIT.md` file. 
You either add it as a file or add it as a link to the license, like this:
```
This project is [MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md
) licensed # as a link

This project is [MIT](./MIT.md) licensed # add a file by adding the `MIT.md` file in the root folder of your `Hello Microverse Project` 

```

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @HtetNaing0814 


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+Acknowledgments
+[Microverse](https://bit.ly/MicroverseTN)
+
+📝 License
+This project is [MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md
+) licensed # as a link
+
+This project is [MIT](./MIT.md) licensed # add a file by adding the `MIT.md` file in the root folder of your `Hello Microverse Project` 
+
+This project is MIT licensed.
```
Comment (zhadier) [inline README.md:41]: - [ ] [_Optional_] Please ensure you use only 1 method suggested by the Previous Reviewer, right now you have 3 statements saying that the ReadMe is MIT licensed.

**Review by zhadier — APPROVED** [review]
## Hi @HtetNaing0814  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


# Congratulations! 🎉 

### Keep Rocking!👏👏👏
_Check the optional suggestions under the Review_

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**